hexa/workspaces/schema/mutations.py

```diff
@@ -114,23 +118,60 @@ def resolve_create_workspace_member(_, info, **kwargs):
         workspace: Workspace = Workspace.objects.filter_for_user(request.user).get(
             slug=input["workspaceSlug"]
         )
-        user = User.objects.get(email=input["userEmail"])
-
-        workspace_membership = WorkspaceMembership.objects.create_if_has_perm(
-            principal=request.user, workspace=workspace, user=user, role=input["role"]
-        )
-        send_mail(
-            title=gettext_lazy(f"You've been added to the workspace {workspace.name}"),
-            template_name="workspaces/mails/invite_member",
-            template_variables={
-                "workspace": workspace.name,
-                "owner": request.user.display_name,
-                "workspace_url": request.build_absolute_uri(
-                    f"//{settings.NEW_FRONTEND_DOMAIN}/workspaces/{workspace.slug}"
+        _user = User.objects.filter(email=input["userEmail"])
```

Comment:
Could you rename `def resolve_create_workspace_member` to the correct mutation name ?

Reply:
I think I would handle this using a try..catch (User.DoesNotExist) block